build.gradle

```diff
@@ -562,6 +563,10 @@ dependencies {
         transitive = false
     }
 
+    if ((usesMixins.toBoolean() || forceEnableMixins.toBoolean()) && stripForgeRequirements.toBoolean()) {
+        runtimeOnlyNonPublishable 'com.cleanroommc:strip-latest-forge-requirements:1.0'
```

Comment:
This dep can just be runtime only? It doesn't need to be compile time?

Reply:
as per the [readme](https://github.com/CleanroomMC/StripLatestForgeRequirements) 
> This is intended purely for dev environments where a mod that you depend on, explicitly depends on a Forge version that's above 2847.
```
dependencies {
    runtimeOnly 'com.cleanroommc:strip-latest-forge-requirements:1.0'
}
```
it should be runtime only and not compile time.